IFC 4.3/Alignment_3/UT_Alignment_3.ifc

```diff
@@ -0,0 +1,62 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [Ifc4X3NotAssigned]'),'2;1');
+FILE_NAME(
+/* name */ 'export\X\5C\X\5CSample_Alignment_RC2.ifc',
+/* time_stamp */ '2020-11-27T10:47:05',
+/* author */ ('stefan'),
+/* organization */ ('Unknown'),
+/* preprocessor_version */ 'GeometryGymIFC v0.1.4.0 by Geometry Gym Pty Ltd built 2020-11-27T09:44:28',
+/* originating_system */ 'GeometryGymIFC v0.1.4.0',
+/* authorization */ 'None');
+
+FILE_SCHEMA (('IFC4X3_RC2'));
+ENDSEC;
+
+DATA;
+#1= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#2= IFCAXIS2PLACEMENT3D(#1,$,$);
+#3= IFCLOCALPLACEMENT($,#2);
+#4= IFCSITE('0Zk6Bd98j1x8h3LY$LYSdo',#10,'sampleSite',$,$,#3,$,$,$,$,$,$,$,$);
+#5= IFCPERSON('stefan','stefan',$,$,$,$,$,$);
+#6= IFCORGANIZATION($,'Unknown',$,$,$);
+#7= IFCPERSONANDORGANIZATION(#5,#6,$);
+#8= IFCAPPLICATION(#9,'0.1.4.0','GeometryGymIFC v0.1.4.0','GeometryGymIFC v0.1.4.0');
+#9= IFCORGANIZATION($,'Geometry Gym Pty Ltd',$,$,$);
+#10= IFCOWNERHISTORY(#7,#8,$,.ADDED.,1606470425,$,$,1606470425);
+#11= IFCPROJECT('1TYUJHHKj64BLuW1RnZUDi',#10,'sampleProject',$,$,$,$,$,#12);
+#12= IFCUNITASSIGNMENT((#13,#14,#15,#16,#17));
+#13= IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#14= IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#15= IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#16= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#17= IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+#18= IFCRELAGGREGATES('2IxncYFtr1GOR0ZHQutL4W',#10,$,$,#11,(#4));
+#19= IFCALIGNMENT('0ScqCY13TCGRKtSY7Hpgra',#10,'DefaultAlignment',$,$,#23,$,#42,$);
+#20= IFCRELCONTAINEDINSPATIALSTRUCTURE('1Z8B3uH5X5lQvcGxLyfi_2',#10,'Site','Site Container for Elements',(#19),#4);
+#21= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#22= IFCAXIS2PLACEMENT3D(#21,$,$);
+#23= IFCLOCALPLACEMENT($,#22);
+#24= IFCCARTESIANPOINT((-29.29264,-6.90761));
+#25= IFCALIGNMENTHORIZONTALSEGMENT($,$,#24,0.462753315190078,0.0,0.0,72.3546021022319,$,.LINE.);
+#26= IFCDIRECTION((0.89482677,0.44641354));
+#27= IFCVECTOR(#26,1.0);
+#28= IFCLINE(#24,#27);
+#29= IFCAXIS2PLACEMENT2D(#24,$);
+#30= IFCCURVESEGMENT(.CONTINUOUS.,#29,IFCNONNEGATIVELENGTHMEASURE(72.3546021022319),#28);
+#31= IFCALIGNMENTVERTICALSEGMENT($,$,0.0,72.349260880448,11.0,-0.0121508006435841,-0.0121508006435841,$,.CONSTANTGRADIENT.);
+#32= IFCDIRECTION((0.99992618,-0.0121505));
+#33= IFCVECTOR(#32,1.0);
+#34= IFCCARTESIANPOINT((72.34926,11.0));
+#35= IFCLINE(#34,#33);
+#36= IFCAXIS2PLACEMENT2D(#34,$);
+#37= IFCCURVESEGMENT(.CONTINUOUS.,#36,IFCNONNEGATIVELENGTHMEASURE(72.3546021022319),#35);
+#38= IFCALIGNMENTHORIZONTAL('2oAQ5oaBr0tOXC7B3AmcuP',#10,$,$,$,$,$,$,(#25));
```

Comment:
please include `#41` as representation for `#38`; similarly for other semantic elements.

Reply:
b4195aa